libraries/AP_HAL_ChibiOS/hwdef/QioTekZealotF427/hwdef.dat

```diff
@@ -0,0 +1,226 @@
+# hw definition file for processing by chibios_hwdef.py
+# for QioTekZealotF427 hardware from Qio-tek.com 
+
+# MCU class and specific type
+MCU STM32F4xx STM32F427xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1021
+
+FLASH_SIZE_KB 2048
+FLASH_RESERVE_START_KB 128
```

Comment:
if you change flash start you'll need to change it here too

Reply:
Jason has told me he wants to keep 128k

Reply:
I'm modified to16K.